docs/src/pages/solid.mdx

```diff
@@ -88,29 +88,26 @@ import { uploadRouter } from "~/server/uploadthing";
 
 export const { GET, POST } = createServerHandler({
   router: uploadRouter,
-  config: {
-    callbackUrl: `http://localhost:${process.env.PORT ?? 3000}/api/uploadthing`,
-  },
 });
 ```
 
-### Creating The UploadThing Components (optional)
+### Creating The UploadThing Components (recommended)
 
-Generating components let's you pass your generic `FileRouter` once and then
-have typesafe components everywhere, instead of having to pass the generic
-everytime you mount a component, but you can also import the components
-individually from `@uploadthing/solid`.
+Generating components lets you pass your generic `FileRouter` and `url` once and
+then have typesafe components everywhere, instead of having to pass the generic
+and url everytime you mount a component, but you can also import the components
```

Comment:
```
Generating components lets you pass your generic `FileRouter` and `url` once and
then have typesafe components everywhere, instead of having to pass them
everytime you mount a component, but you can also import the components
```